src/compiler/types.ts

```diff
@@ -1008,7 +1016,6 @@ export type ForEachChildNodes =
 /** @internal */
 export type VisitEachChildNodes =

```

Comment:
could we just replace this type with `HasChildren` now?

Reply:
This was resolved without comment; do we still need this type? (I don't care either way, I was just interested in the rationale to leave it.)

Reply:
Oops, removed the reference, forgot to remove the type.